schemas/dab.draft.schema.json

```diff
@@ -168,6 +168,11 @@
               "description": "Sets the default number of records returned in a single response. When this limit is reached, a continuation token is provided to retrieve the next page. If set to null, the default value is 100.",
```

Comment:
Description says:

`When Data API Builder (DAB) is deployed`

-> Builder should be lower cased as in -> `builder`